tests/k8s/tests/01-guestbook-test.sh

```diff
@@ -3,6 +3,7 @@
 # This tests:
 # - Kubernetes network policy enforcement in the same namespace
 # - Kubernetes services translation to backend IP
+# - CNP with the same name on different namespaces
```

Comment:
Is there a need for this to be converted to a Ginkgo test as well?

Reply:
Yes. I'll focus on the ginkgo tests this week for all past PRs.

Reply:
Can you file a GH issue so this conversion is tracked?